masterlist.yaml

```diff
@@ -14609,6 +14619,12 @@ plugins:
         crc: 0x7E36F0E5
         util: '[SSEEdit v4.0.1](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 122
+  - name: 'ClockworkCourierCounterspell.esp'
+    group: "Fixes & Resources"
+    clean:
+      - crc: 0x6abc5154
+        util: "SSEEdit v4.0.3"
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087/' ]
```

Comment:
Since all these files are on the same page, they can be placed together for easier maintenance.